src/components/element-editor/conditional-rendering/conditonal-rendering.tsx

```diff
@@ -0,0 +1,120 @@
+import { useCallback, useEffect, useState } from "react";
+import { MultiEntry } from "src/components/common";
+import { IConditionalRendering, useBuilder } from "src/context-providers";
+import { ConditionalRenderingChild } from "./conditional-rendering-child";
+
+interface IOptions {
+    label?: string;
+    id?: string;
+}
+
+export const ConditionalRendering = () => {
+    // =========================================================================
+    // CONST, STATE, REFS
+    // =========================================================================
+
+    const { focusedElement, elements } = useBuilder();
+    const element = focusedElement.element;
+    const [childEntryValues, setChildEntryValues] =
+        useState<IConditionalRendering[]>();
+
+    // =====================================================================
+    // HELPER FUNCTIONS
+    // =====================================================================
+
+    const getElementOption = useCallback(() => {
+        const options: IOptions[] = [];
+        Object.entries(elements).forEach(([key, value]) => {
+            if (key !== element.internalId) {
+                const optionItem = {
+                    label: value.label,
+                    id: value.id,
+                };
+                options.push(optionItem);
+            }
+        });
+        return options;
+    }, [elements, element.internalId]);
```

Comment:
- rename as `getElementOptions` because it can return more than one option
- `useCallback` is not necessary because the function will still get called on every render
- `useCallback` is usually needed if a function is used as a dependency in `useEffect` or other `useCallback`, else there is little to no savings 

```suggestion
    const getElementOptions = () => {
        const options: IOptions[] = [];
        Object.entries(elements).forEach(([key, value]) => {
            if (key !== element.internalId) {
                const optionItem = {
                    label: value.label,
                    id: value.id,
                };
                options.push(optionItem);
            }
        });
        return options;
    };
```